static/js/settings_account.js

```diff
@@ -646,7 +646,6 @@ export function set_up() {
         $("body").one("hidden.bs.modal", "#user-profile-modal", (e) => {
             e.preventDefault();
             e.stopPropagation();
-            popovers.hide_user_profile();
```

Comment:
I see you comment in the commit message why you removed this, but can you explain why removing this line won't re-create whatever problem motivated this being here?  (Or did it never make sense to have this call?)

Reply:
I think calling `hide_user_profile` was not required here initially itself. This is because, '.hide()` which was being called in `hide_user_profile` returns before the `hidden.bs.modal` event is being triggered. So, basically modal has already been hidden if `hidden.bs.modal` event has been triggered so we do not need to call `.hide()` again.

Reply:
Yeah, that's correct.